### DISCUSSION TITLE
Query Priority?

### DISCUSSION THREAD

**Discussion by kershnerd** [TanStack/query#6080] [General]
I have an unfortunate app that needs to make over 100 API calls spread out across many different components.  React-query handles this really well and I am impressed.

Some of those requests get stalled for a few seconds because I believe Chrome is telling me that I have hit the limit of the TCP.

Is there any way to specify a priority so that queries get submitted and executed in priority order?  Or anything like this?

**Comment by TkDodo** [comment]
no, you'd need to handle this on your api layer. You probably want some sort of task queue, maybe something like swimmer?

https://github.com/tannerlinsley/swimmer
